### PR TITLE
Added text to the login page

### PR DESCRIPTION
**Pull request by 1eob**
Added a simple temporary fix for the register button not working by adding instructions on how to register on the login page

image

### CODE REVIEW

File: ui/modules/multiplayer/login.partial.html [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
+			<img src="local://local/ui/modules/multiplayer/beammp.png" alt="BeamMP Logo">
+			<h2 class="text-red" id="LOGINERRORFIELD"></h2>
+			<form id="LoginContainer">
+				<h2 class="text-white" id="loginHeader">{{:: 'ui.multiplayer.loginDescription1' | translate}}</h2>
+				<input (keyup.enter)="multiplayer.login()" type="text" id="loginUsername" placeholder="Username / Email"/>
+				<input (keyup.enter)="multiplayer.login()" type="password" id="loginPassword" placeholder="Password"/>
+				<div style="display:flex; align-content: space-evenly;">
+					<md-button class="button" ng-click="login()">{{:: 'ui.multiplayer.login' | translate}}</md-button>
+					<md-button class="button" ng-click="switchConnection()">{{:: 'ui.multiplayer.playAsGuest' | translate}}</md-button>
+					<md-button class="button" onclick="openExternalLink('https://forum.beammp.com/signup')">{{:: 'ui.multiplayer.register' | translate}}</md-button>
```
Comment (OfficialLambdax) [inline ui/modules/multiplayer/login.partial.html:14]: What happens if you comment out the Register Button?

**Review by OfficialLambdax — CHANGES_REQUESTED** [review]

File: ui/modules/multiplayer/login.partial.html (hunk 1 above)
Comment (1eob) [reply ui/modules/multiplayer/login.partial.html:14]: I mean i can certainly try. im still heavily learning so i dont want to do something that might break another thing

Comment (1eob) [reply ui/modules/multiplayer/login.partial.html:14]: Actually it looks fine the ui has seem to adjust automatically 

image

**Review by OfficialLambdax — APPROVED** [review]

File: ui/modules/multiplayer/login.partial.html (hunk 1 above)
Comment (OfficialLambdax) [reply ui/modules/multiplayer/login.partial.html:14]: push that change to your fork pls

Comment (1eob) [reply ui/modules/multiplayer/login.partial.html:14]: Done

**Review by OfficialLambdax — APPROVED** [review]

**Review by Starystars67 — APPROVED** [review]
This does indeed work as a stop-gap until the EA launcher moves to public.

Thank you @1eob

**Comment by 1eob** [comment]
> This does indeed work as a stop-gap until the EA launcher moves to public.
> 
> Thank you @1eob

Np ik it's small but it's a start for looking into the code

**Comment by Starystars67** [comment]
> > This does indeed work as a stop-gap until the EA launcher moves to public.
> > Thank you @1eob
> 
> Np ik it's small but it's a start for looking into the code

I completely agree :)

**Pull request merged**